tests/test_arr_venue.py

```diff
@@ -1092,3 +1092,46 @@ def test_ethics_review_stage(self, venue, client, helpers):
         assert client.get_invitation('aclweb.org/ACL/ARR/2021/September/Paper3/-/Ethics_Review')
         assert client.get_invitation('aclweb.org/ACL/ARR/2021/September/Paper4/-/Ethics_Review')               
         assert client.get_invitation('aclweb.org/ACL/ARR/2021/September/Paper5/-/Ethics_Review')
+   
+    def test_comment_email_pcs(self, venue, client, helpers, test_client): 
+        pc_client=openreview.Client(username='pc@aclrollingreview.org', password='1234')
+        super_client=openreview.Client(username='openreview.net', password='1234')
```

Comment:
why do you need to create super_client and not using client instead?

Reply:
When the tests were failing I wanted to make sure I was using the super user client. I just changed it back